packages/runtime/test/lib/testUtils.ts

```diff
@@ -729,8 +731,38 @@ export async function executeFullRequestAndShareThirdPartyRelationshipAttributeF
         return e.data.request.id === localRequest.id && e.data.newStatus === LocalRequestStatus.Completed;
     });
 
-    const ownSharedThirdPartyRelationshipAttribute = (await owner.consumption.attributes.getAttribute({ id: sharedAttributeId })).value;
-    return ownSharedThirdPartyRelationshipAttribute;
+    const ownSharedRelationshipAttribute = (await owner.consumption.attributes.getAttribute({ id: sharedAttributeId })).value;
+    return ownSharedRelationshipAttribute;
+}
+
+export async function executeFullShareAndAcceptAttributeRequestFlow(
+    owner: TestRuntimeServices,
+    peer: TestRuntimeServices,
+    requestItem: ShareAttributeRequestItem
+): Promise<LocalAttributeDTO> {
+    const request = Request.from({
+        items: [requestItem]
+    });
+
+    const canCreateResult = await owner.consumption.outgoingRequests.canCreate({
+        content: request.toJSON(),
+        peer: peer.address
+    });
+
+    expect(canCreateResult.value.isSuccess).toBe(true);
+
+    const createRequestResult = await owner.consumption.outgoingRequests.create({
+        content: request.toJSON(),
+        peer: peer.address
+    });
+
+    await owner.transport.messages.sendMessage({
+        recipients: [peer.address],
+        content: createRequestResult.value.content
+    });
+
+    const result = await acceptIncomingShareAttributeRequest(owner, peer, createRequestResult.value.id);
+    return result;
```

Comment:
Please use a more descriptive name, such that it is clear what is returned, e.g. `ownSharedAttribute`.